pymemcache/pool.py

```diff
@@ -35,6 +36,7 @@ def __init__(self, obj_creator,
             self._lock = lock_generator()
         self._after_remove = after_remove
         max_size = max_size or 2 ** 31
+        self.obj_idle_timeout = obj_idle_timeout or 60
```

Comment:
This means that both `None` and `0` signal the pool that the default timeout (`60`) should be used. Is that intended?